frontend/lib/routes/meeting/AddMeetingMemberForm.dart

```diff
@@ -0,0 +1,197 @@
+              return results[index];
+            }));
+  }
+
+  void _getMemberData(String id, String name) {
+    _memberID = id;
+    _memberName = name;
+    _searchMembersController.text = name;
+    disappearSearchResults = true;
+    setState(() {});
+  }
+
+  List<Widget> getListCards(List<Member> members) {
+    List<Widget> results = [];
+    if (members.length != 0) {
+      results.add(getDivider("Members"));
+      results.addAll(members.map((e) => SearchResultWidget(
+            member: e,
+            getMemberData: _getMemberData,
+          )));
+    }
+    return results;
+  }
+
+  Widget getDivider(String name) {
+    return Card(
+        margin: EdgeInsets.zero,
+        child: Column(
+          children: [
+            Container(
+              child: Text(name, style: TextStyle(fontSize: 18)),
+              margin: EdgeInsets.fromLTRB(0, 8, 0, 4),
+            ),
+          ],
+        ));
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    List<String> types = ["Member", "Company Representative"];
+    return Form(
+        key: _formKey,
+        child: Column(
+          children: [
+            Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: TextFormField(
+                    controller: _searchMembersController,
+                    validator: (value) {
+                      if (value == null || value.isEmpty) {
+                        return 'Please enter a member';
+                      }
+                      return null;
+                    },
+                    decoration: const InputDecoration(
+                        icon: const Icon(Icons.groups), labelText: "Member *"),
+                    onChanged: (newQuery) {
+                      setState(() {});
+                      if (_searchMembersController.text.length > 1) {
+                        this.membs = _memberService.getMembers(
+                            name: _searchMembersController.text);
+                      }
+                    })),
+            ...getResults(MediaQuery.of(context).size.height * 0.3),
+            Padding(
+              padding: const EdgeInsets.all(8.0),
+              child: DropdownButtonFormField<String>(
+                icon: Icon(Icons.tag),
+                items: types
+                    .map((e) =>
+                        DropdownMenuItem<String>(value: e, child: Text(e)))
+                    .toList(),
+                value: types[0],
+                selectedItemBuilder: (BuildContext context) {
+                  return types.map((e) {
+                    return Align(
+                      alignment: AlignmentDirectional.centerStart,
+                      child: Container(child: Text(e)),
+                    );
+                  }).toList();
+                },
+                onChanged: (next) {
+                  setState(() {
+                    if (next == "Company Representative") {
+                      type = "COMPANYREP";
```

Comment:
Maybe we should have an enum for the types as well, like we do for the roles (https://github.com/sinfo/deck2/blob/master/frontend/lib/services/authService.dart#L14). What do you think?